src/scripts.js

```diff
@@ -7,59 +7,62 @@ import Trip from './trip'
 import TripsRepository from './trips-repo'
 import { fetchTravelers, fetchSingleTraveler, fetchTrips, fetchDestinations }
   from './apiCalls';
+import pageLoadDom from './domUpdates'
 import './css/base.scss';
 import './images/turing-logo.png'
 
 
 // Global Variables
 
-const today = '2020/04/09'
+const today = '2020/04/03'
 let currentTraveler
 let tripRepo
 
 console.log('This is the JavaScript entry file - your code begins here.');
 
-// Query Selectors
 
-const headerMessage = document.querySelector('#splashMessage')
 
 
 // Functions
 
-const pageLoad = () => {
-  fetchData()
 
-}
 
-const fetchData = () => {
+const pageLoad = () => {
   Promise.all([fetchSingleTraveler, fetchTrips, fetchDestinations]).then(values => {
```

Comment:
Man, I'm glad I'm not the only one who fell back on Promise.all again. It was so dang hard to make sure all the API calls resolved otherwise!